docs/source/tutorial_metrics.md

```diff
@@ -0,0 +1,372 @@
+# Understanding and adding new metrics
+
+Author: Stephen Roller
+
+## Introduction and Standard Metrics
+
+ParlAI contains a number of built-in metrics that are automatically computed when
+we train and evaluate models. Some of these metrics are _text generation_ metrics,
+which happen any time we generate a text: this includes F1, BLEU and Accuracy.
```

Comment:
Like you did below, maybe after introducing the metrics here, say that more info about them is below. Also, I think it catches the eye better (if you are glancing the doc) to have these bold.

Reply:
You mean bold F1, BLEU, and Accuracy?